linux_sql/README.md

```diff
@@ -1,2 +1,84 @@
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+
+#Table of Contents
```

Comment:
https://github.com/jarviscanada/jarvis_data_eng_adams/blob/4f9df27ef7fcf7729bb1272999bbff5fe3a62dae/linux_sql/README.md

double check the rendered markdown file and fix syntax issue